en-devonfw.dict

```diff
@@ -1206,3 +1201,30 @@ jre
 propertymerge
```

Comment:
Delete?

Reply:
These words are not handled in backticks etc should not be part of dict